plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -169,6 +175,7 @@
 
 public class TrinoS3FileSystem
         extends FileSystem
+        implements FileSystemWithBatchDelete
```

Comment:
Aren't we moving away from use of hadoop's `FileSystem` towards `TrinoFileSystem`. Should `TrinoFileSystem` support batch delete?